app/controllers/discover.js

```diff
@@ -235,6 +298,15 @@ export default ApplicationController.extend({
 
         clearFilters() {
             this.set('facetFilters', Ember.Object.create());
+            let params = this.get('filterQueryParams');
+            for (var param in params) {
+                let key = params[param];
+                if (params.indexOf(key) > -1) {
+                    this.set(key, '');
```

Comment:
Should this be `null`? When I clear a facet, it stays in the URL, e.g. `?contributors=&sources=`